.github/workflows/release.yml

```diff
@@ -17,8 +17,6 @@ jobs:
           fetch-depth: 0
       - name: Setup Nodejs Env
         run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
-      - name: Install dependencies
-        run: npm ci
```

Comment:
The only dependencies in this repo are peer dependencies right now (installed by the consumer). Running `npm ci` isn't really necessary.